docs/architecture/adr-076-sign-mode-unified.md

```diff
@@ -0,0 +1,327 @@
+# ADR 075: Sign Mode Unified
+
```

Comment:
What does this sign mode unify? I think both Amino JSON and Textual are designed to be self-descriptive documents. The big selling point of this is using on-chain state to mitigate the limitations of the other approach. I would use a great new name for great new tech.